packages/account/src/Components/poi/status/unsupported/card-details/documents-upload.jsx

```diff
@@ -0,0 +1,130 @@
+    const { fields, documents_title, documents } = data;
+
+    const fields_title = localize('First, enter your {{label}} and the expiry date.', {
+        label: fields[0].label,
+    });
+
+    const goBack = () => {
+        goToCards();
+    };
+
+    return (
+        <div
+            className={classNames(ROOT_CLASS, {
+                [`${ROOT_CLASS}--mobile`]: isMobile(),
+            })}
+        >
+            <Formik
+                initialValues={initial_values || setInitialValues([...fields, ...documents])}
+                validate={values => validateFields(values, fields, documents)}
+                onSubmit={onSubmit}
+            >
+                {({ values, isValid, touched }) => {
+                    const is_form_touched = Object.keys(touched).length > 0;
+                    const is_form_empty = Object.values(values).some(
+                        (field, key) => (field === null || field === '') && fields[key]?.required
+                    );
+
+                    return (
+                        <Form className={`${ROOT_CLASS}__form`}>
+                            <div className={`${ROOT_CLASS}__fields-content`}>
+                                <Text as='h3' size='s' color='prominent'>
+                                    {fields_title}
+                                </Text>
+                                <div className={`${ROOT_CLASS}__fields-wrap`}>
+                                    {fields.map(field => (
+                                        <InputField key={field.name} data={field} />
+                                    ))}
+                                </div>
+                                <div className={`${ROOT_CLASS}__divider`} />
+                                <Text as='h3' size='s' color='prominent'>
+                                    {documents_title}
+                                </Text>
+                                <div className={`${ROOT_CLASS}__uploaders-wrap`}>
+                                    {documents.map(item => (
+                                        <Uploader
+                                            key={item.name}
+                                            data={item}
+                                            value={values[item.name]}
+                                            is_full={documents.length === 1}
+                                        />
+                                    ))}
+                                </div>
+                                <div className={`${ROOT_CLASS}__icons`}>
+                                    {icons.map(item => (
+                                        <IconsItem key={item.icon} data={item} />
+                                    ))}
+                                </div>
+                            </div>
+                            <div className={`${ROOT_CLASS}__btns`}>
+                                <Button
+                                    onClick={goBack}
```

Comment:
Why not just call `goToCards` directly since that's all the `goBack` function does

```suggestion
                                    onClick={goToCards}
```

Reply:
Thanks, I removed the extra function🙏🏻